mods/ra/maps/survival02/survival02.lua

```diff
@@ -98,10 +99,18 @@ Tick = function()
 		end
 	end
 
-	if DateTime.Minutes(5) == TimerTicks - DateTime.GameTime then
+	if DateTime.Minutes(5) == ticked then
 		Media.PlaySpeechNotification(allies, "WarningFiveMinutesRemaining")
 		InitCountDown()
 	end
+
+	if 0 < ticked then
```

Comment:
Convention is to put constants (i.e. the 0) on the right hand side of the expression